src/System.Private.CoreLib/shared/System/Runtime/InteropServices/WindowsRuntime/EventRegistrationToken.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices.WindowsRuntime
+{
+    // Event registration tokens are 64 bit opaque structures returned from WinRT style event adders, in order
+    // to signify a registration of a particular delegate to an event.  The token's only real use is to
+    // unregister the same delgate from the event at a later time.
+    public struct EventRegistrationToken : IEquatable<EventRegistrationToken>
+    {
+        internal readonly ulong _value;
+
+        internal EventRegistrationToken(ulong value) => _value = value;
```

Comment:
ProjectN needs the `ctor` & `Value` to be public. They are called from `System.Private.Interop`.

Reply:
Ok